app/main.py

```diff
@@ -1,8 +1,51 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self: None, comfort_class: None,
+                 clean_mark: None,
+                 brand: None) -> None:
```

Comment:
Change your type annotation to indicate what types you expect to obtain for each argument, they should not be `None`.